servers/visual/shader_language.cpp

```diff
@@ -3077,7 +3077,44 @@ Error ShaderLanguage::_parse_block(BlockNode *p_block, const Map<StringName, Dat
 				_set_tkpos(pos); //rollback
 			}
 
-		} else {
+		} else if (tk.type == TK_CF_RETURN) {
+            FunctionNode *function = NULL;
```

Comment:
Fix yer spaces. We use tabs around here. (you can run clang-format for a semi-automated fix, AFAIK)